### PR TITLE
[UI/UX:HelpQueue] Office hour staff text color

### PR DESCRIPTION
**Pull request by billyvande**
### What is the current behavior?
Text color is different in dark mode. 

### What is the new behavior?
Since the objective is to make text color same. Now, office hour text has same color in dark mode.

### CODE REVIEW

**Review by cjreed121 — CHANGES_REQUESTED** [review]
Rather than adding hard coded CSS. We should be able to match up the classes on both. You'll see that one of the labels has classes `btn` and `filter-buttons` while the other just has `btn`. Let's keep it consistent so they both have `btn` and `filter-buttons`.

**Comment by billyvande** [comment]
Thanks for corrections. This is my first time contributing to an open source. 
I update Office hours header and match the class.

File: site/app/templates/officeHoursQueue/QueueHeader.twig [hunk 1]
```diff
@@ -27,7 +27,7 @@
         {% for queue in viewer.getAllQueues() %}
             {% if queue['open'] != 0 %}
 
-                <label tabindex="0" class="btn filter-buttons"  style="word-wrap: break-word; border-color: {{ viewer.getColorFromCode(queue['code']) }}; background: {{ viewer.getColorFromCode(queue['code']) }};">{{queue['code']}}
+                <label tabindex="0" class="btn filter-buttons"  style="word-wrap: break-word; border-color: {{ viewer.getColorFromCode(queue['code']) }}; background: {{ viewer.getColorFromCode(queue['code']) }}; color: inherit;">{{queue['code']}}
```
Comment (cjreed121) [inline site/app/templates/officeHoursQueue/QueueHeader.twig:30]: Can you remove `color: inherit` here

File: site/app/templates/officeHoursQueue/QueueHeader.twig [hunk 2]
```diff
@@ -45,7 +45,7 @@
         {% for queue in viewer.getAllQueues() %}
             {% if queue['open'] == 0 %}
 
-                <label tabindex="0" class="btn"  style="word-wrap: break-word; border-color: {{ viewer.getColorFromCode(queue['code']) }}; background: {{ viewer.getColorFromCode(queue['code']) }}; color: inherit;">{{queue['code']}}
+                <label tabindex="0" class="btn filter-buttons"  style="word-wrap: break-word; border-color: {{ viewer.getColorFromCode(queue['code']) }}; background: {{ viewer.getColorFromCode(queue['code']) }}; color: inherit;">{{queue['code']}}
```
Comment (cjreed121) [inline site/app/templates/officeHoursQueue/QueueHeader.twig:48]: Also remove here too

**Review by cjreed121 — CHANGES_REQUESTED** [review]
I used WAVE to check accessibility and there was some contrast errors with this. If you can quickly make the changes I've noted it should fix them and I think this is then good to merge.

File: site/app/templates/officeHoursQueue/QueueHeader.twig (hunk 1 above)
Comment (billyvande) [reply site/app/templates/officeHoursQueue/QueueHeader.twig:30]: yes, sure!

**Comment by billyvande** [comment]
Before : 
image
After : 
image

I wish my PR solve the issues.

**Comment by cjreed121** [comment]
Please leave the title as is so it complies with https://submitty.org/developer/getting_started/how_to_contribute

**Pull request merged**